### PR TITLE
Add recipe for augeas

### PR DESCRIPTION
**Pull request by birdsarah**

### CODE REVIEW

**Comment by msarahan** [comment]
Not clear to me why things are failing here.  I don't know where that unicode folder of headers is expected to come from.  I recommend developing and testing your build locally with the conda-forge docker container:

https://quay.io/repository/condaforge/linux-anvil

```docker run -ti quay.io/condaforge/linux-anvil (plus any volume mounts you want to do)```

**Comment by birdsarah** [comment]
@msarahan, would appreciate any help / thoughts you have - i can't reproduce this locally.

**Comment by birdsarah** [comment]
i take it back - i can reproduce locally!

**Comment by birdsarah** [comment]
@msarahan - feedback welcome, not sure why appveyor's trying to run - i'm trying to skip windows but i probably don't understand how all the config works

**Comment by birdsarah** [comment]
@msarahan - can this be merged?

File: recipes/augeas/build.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+export CFLAGS=-I$PREFIX/include/
+export LDFLAGS=-L$PREFIX/lib/
+
+if [ `uname` == Darwin ]; then
+  # OSX only
+  # If these flags are set on linux build, there's a linking error
```
Comment (jakirkham) [inline recipes/augeas/build.sh]: Do you have an example build log for this?

File: recipes/augeas/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - libxml2
+    - readline
+    - icu
```
Comment (jakirkham) [inline recipes/augeas/meta.yaml]: Please pin according to the [pinning script]( https://github.com/conda-forge/conda-forge.github.io/blob/346dd29465bd97f26baa5427abe29d3476ff9ffa/scripts/pin_the_slow_way.py#L32-L73 ). Also aren't these needed at run time too?

File: recipes/augeas/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+test:
+  commands:
+    - test -e $PREFIX/include/augeas.h
```
Comment (jakirkham) [inline recipes/augeas/meta.yaml]: Are there any libraries or binaries that we can test for?

Comment (jakirkham) [inline recipes/augeas/meta.yaml]: Running `--help` on CLI tools is a common thing to do.

File: recipes/augeas/build.sh [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  export LIBXML_CFLAGS=-I$PREFIX/include/libxml2/
+  export LIBXML_LIBS=-L$PREFIX/lib/
+fi
+./configure --prefix=$PREFIX
+make
```
Comment (jakirkham) [inline recipes/augeas/build.sh]: Is there a `make check` that can be run?

**Comment by scopatz** [comment]
pinging this recipe!

**Pull request closed without merging**